app/models/user.rb

```diff
@@ -215,6 +115,29 @@ def validate_pin_pattern
   end
 
 
+  # Another piece of code has determined this user is all set to be called
+  # complete and done.  Note: Usually, expect the next step to be the calling of
+  # Patron Service to create the user in Sierra.
+  def save_as_complete!
+    self.status = STATUS_LABELS['complete']
+    self.save
```

Comment:
There seems to be two versions of this method — one with a `!` and one at the end. Given Rails has `save` and `save!` this doesn't seem unreasonable, but I would definitely use the `save!` method in `save_as_complete!` and `save_as_pending!`, and `save` in `save_as_complete` and `save_as_pending`. Or just go with one version, but use the matching Rails `save`. Also I'd move the code together in the file.

Reply:
Good catch, looks like a merge error.  Corrected.